### PR TITLE
[ADD] account_tax_supported module

### LINKED ISSUE #109: [RFC] Supported Taxes on Invoices
The requirement is to compute taxes on Invoices, that are not charged to the customer, and will be supported by the seller.

The plan to support this is:

- On Account Taxes, add a flag to mark supported tax codes.
- On invoice Lines

  - Add a "Supported Tax Amount" column
  - The supported Taxes should be added to the "Supported Tax Amount" columns and not to the "Total Amount Including Taxes" column
- The Invoice total amount due should not inclode the supported tax amounts
- The accounting moves should be created for the supported taxes, and a debit a credit move needs to be made. Note that the receivable move line amount should not include the supported taxes.

### PR DESCRIPTION
**Pull request by dreispt**
Closes #109

### CODE REVIEW

**Review by mgosai — APPROVED** [review]

**Comment by dreispt** [comment]
@OCA/accounting-maintainers This one just needs one more push... Thanks!

File: account_tax_expensed/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                )
+                res.append(values2)
+        return res
+
+
+class AccountInvoiceTax(models.Model):
+
+    _inherit = "account.invoice.tax"
+
+    amount_tax_expense = fields.Monetary(string="Tax Expense")
```
Comment (max3903) [inline account_tax_expensed/models/account_invoice.py:52]: @dreispt Separate file?

File: account_tax_expensed/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    "type": "tax",
+                    "name": tax_line.name,
+                    "price_unit": tax_line.amount_tax_expense,
+                    "quantity": 1,
+                    "price": tax_line.amount_tax_expense,
+                    "account_id": tax_line.account_id.id,
+                    "account_analytic_id": tax_line.account_analytic_id.id,
+                    "analytic_tag_ids": analytic_tag_ids,
+                    "invoice_id": self.id,
+                }
```
Comment (max3903) [inline account_tax_expensed/models/account_invoice.py]: @dreispt `prepare_...` method to return this dictionary?

File: account_tax_expensed/views/account_invoice.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline account_tax_expensed/views/account_invoice.xml]:
```suggestion
```

File: account_tax_expensed/views/account_tax.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline account_tax_expensed/views/account_tax.xml]:
```suggestion
```

**Review by max3903 — APPROVED** [review]
Minor changes

File: account_tax_expensed/models/account_invoice.py (hunk 1 above)
Comment (dreispt) [reply account_tax_expensed/models/account_invoice.py:52]: This is tightly related to Invoice Lines - it is for a to-many "taxes" field in invoice lines.
So IMO it is best kept in the same file.

File: account_tax_expensed/models/account_invoice.py (hunk 2 above)
Comment (dreispt) [reply account_tax_expensed/models/account_invoice.py]: OK

**Comment by dreispt** [comment]
I addressed review comments.
It should be good to merge.
Thank you.

**Comment by dreispt** [comment]
Development discontinued

**Pull request closed without merging**